editor/filesystem_dock.cpp

```diff
@@ -4213,6 +4218,22 @@ FileSystemDock::FileSystemDock() {
 	file_list_search_box->connect(SceneStringName(text_changed), callable_mp(this, &FileSystemDock::_search_changed).bind(file_list_search_box));
 	path_hb->add_child(file_list_search_box);
 
+	thumbnail_scroll_container = memnew(VBoxContainer);
+	thumbnail_scroll_container->set_alignment(ALIGNMENT_CENTER);
+	thumbnail_scroll_container->set_tooltip_text("Change Thumbnail Size");
+	path_hb->add_child(thumbnail_scroll_container);
+
+	file_list_thumbnail_scroll = memnew(HScrollBar);
+	file_list_thumbnail_scroll->set_step(4);
+	file_list_thumbnail_scroll->set_custom_minimum_size(Vector2(40, 10) * EDSCALE);
+	file_list_thumbnail_scroll->set_max(128);
+	file_list_thumbnail_scroll->set_min(16);
+	file_list_thumbnail_scroll->set_value(EDITOR_GET("docks/filesystem/thumbnail_size"));
+	file_list_thumbnail_scroll->set_min(16);
+	thumbnail_scroll_container->add_child(file_list_thumbnail_scroll);
+	file_list_thumbnail_scroll->hide();
+	file_list_thumbnail_scroll->connect("value_changed", callable_mp(this, &FileSystemDock::_update_thumbnail_size).unbind(1).bind(file_list_thumbnail_scroll->get_value()));
```

Comment:
```suggestion
	file_list_thumbnail_scroll->connect("value_changed", callable_mp(this, &FileSystemDock::_update_thumbnail_size));
```
Pretty sure this does nothing. It wouldn't be working correctly if it did (you bound a constant value).

Reply:
Interesting. I think I might've misunderstood and thought it bound a reference rather than a constant value. Even weirder that it works despite it being there...